### ISSUE TITLE
Arbitrary argument lists should resolve to tuple type

### ISSUE THREAD

**Issue by NateScarlet** [microsoft/pyright#396]
**Describe the bug**
Arbitrary argument lists resolved to List type.

**To Reproduce**
Use `*args`

**Expected behavior**
`args` resolve to tuple type

**Screenshots or Code**
```python
import typing


def args_as_tuple(*args: str) -> typing.Tuple[str, ...]:
    return args
```
![image](https://user-images.githubusercontent.com/21083014/70295019-6920d180-1820-11ea-9e6b-36dfab32572d.png)

```shell
$ py -2.7 -c "from __future__ import print_function;(lambda *args: print(type(args)))()"
<type 'tuple'>
$ py -3.7 -c "from __future__ import print_function;(lambda *args: print(type(args)))()"
<class 'tuple'>
```

**VS Code extension or command-line**
VSCode extension: 1.1.7

**Additional context**
https://docs.python.org/3/tutorial/controlflow.html#arbitrary-argument-lists

**Comment by erictraut** [comment]
Thanks, good catch. I added a fix along with a unit test.

**Comment by erictraut** [comment]
This is now implemented in version 1.1.9, which I just published.
